app/views/layouts/_doorbell.html.erb

```diff
@@ -0,0 +1,20 @@
+<!-- Doorbell -->
+<script type="text/javascript">
+    window.doorbellOptions = {
+        "id": '6873',
+        "hideEmail": false,
+        "email": '<%= current_user.present? ? current_user.email : "" %>',
```

Comment:
If you wanted to shorten these ternary expressions and keep the templates a little leaner, you could use safe navigation to guard against no `current_user`

```
<%= current_user.present? ? current_user.email : "" %>

<%= current_user&.email %>
```

Reply:
neat - I hadn't seen that ternary syntax before.